api-report/container-loader.api.md

```diff
@@ -93,12 +93,12 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
     // (undocumented)
     proposeCodeDetails(codeDetails: IFluidCodeDetails): Promise<boolean>;
     raiseContainerWarning(warning: ContainerWarning): void;
-    // @deprecated
-    get readonly(): boolean | undefined;
     // (undocumented)
+    /**
```

Comment:
these docs are autogenerated; if you need to change anything here it should be done through the original source

Reply:
This change was made automatically after running the build on my local machine. Is this not the correct procedure?

Reply:
yeah that's right. there wasn't any comment added in the source in this pr though 🤔